tests/test_rootfs.py

```diff
@@ -204,6 +192,54 @@ def test_expected_files_ext234_mender_connect(
                 assert "ShellCommand" in mender_connect_vars, mender_connect_vars
                 assert "User" in mender_connect_vars, mender_connect_vars
 
+    @pytest.mark.only_with_image("ext4", "ext3", "ext2")
+    @pytest.mark.min_mender_version("2.6.0")
+    def test_expected_files_ext234_mender_configure(
+        self, conversion, bitbake_path, bitbake_variables, latest_rootfs
+    ):
+        """Test mender-configure expected files"""
+
+        # Expect to be installed for Yocto and not for mender-convert
+        expect_installed = not conversion
+
+        with make_tempdir() as tmpdir:
+
+            # Check whether mender-configure exists in  /usr/share/mender/modules/v3
+            self.verify_file_exists(
+                tmpdir,
+                latest_rootfs,
+                "/usr/share/mender/modules/v3",
+                "mender-configure",
+                expect_installed,
+            )
+
+            # Check whether mender-configure is executable
+            if expect_installed:
+                self.verify_file_executable(
+                    tmpdir,
+                    latest_rootfs,
+                    "/usr/share/mender/modules/v3",
+                    "mender-configure",
+                )
+
+            # Check whether mender-inventory-mender-configure exists in  /usr/share/mender/inventory
+            self.verify_file_exists(
+                tmpdir,
+                latest_rootfs,
+                "/usr/share/mender/inventory",
+                "mender-inventory-mender-configure",
+                expect_installed,
+            )
+
+            # Check whether mender-inventory-mender-configure is executable
+            if expect_installed:
+                self.verify_file_executable(
+                    tmpdir,
+                    latest_rootfs,
+                    "/usr/share/mender/inventory",
+                    "mender-inventory-mender-configure",
+                )
```

Comment:
We should check (and implement) two more things here: That `/var/lib/mender-configure` is a symlink which points to `/data/mender-configure`, and that the latter exists.

Reply:
I don't see a test for either of these two things... Verifying `/data/mender-configure` needs to be in `test_part_image.py`, I suppose.

Reply:
Sorry! You are right.

I implemented it but forgot to test. I'll extend the test now and create a new meta-mender PR to verify.

Reply:
> I don't see a test for either of these two things... [,,,]

Well, we check that `/var/lib/mender-configure` exists  only, not that it is a symlink. See https://github.com/mendersoftware/mender-image-tests/pull/39/files#diff-61f29111cea287297ab544f795cf48c3a8f83c2b5a24628fef969b8e65215772R243

Reply:
I know. But it's important that it's a symlink! Important enough that we should test it I think. :-)